src/emqx.appup.src

```diff
@@ -3,9 +3,11 @@
 {VSN,
   [{"4.3.17",
     [{load_module,emqx_shared_sub,brutal_purge,soft_purge,[]},
+     {load_module,emqx_broker_sup,brutal_purge,soft_purge,[]},
```

Comment:
Q: should we add `{update, emqx_broker_sup, supervisor}`, as its start specification has been changed?

Reply:
I didn't know there is a `{update, emqx_broker_sup, supervisor}` to replace the specs.
always thought it would be a complicated upgrade procedure for supervisors.

if it's as simple as that, we should definitely add it.
otherwise this is not really a big deal to leave the old spec running.